tcf_website/templates/base/bugform.html

```diff
@@ -0,0 +1,50 @@
+{% load static %}
+
+{% block styles %}
+<link rel="stylesheet" href="{% static 'common/form.css' %}" />
+{% endblock %}
+
+<!-- found a bug form pop up (modal) -->
+<div class="modal fade" id="bugModal" tabindex="-1" role="dialog" aria-labelledby="bugModal" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="bugModal">Report a Bug </h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <form class="needs-validation" id="bugform" novalidate>
+          <div class="form-row">
+              <label for="urlField">URL</label>
+              <input type="text" class="form-control" id="urlField" placeholder="thecourseforum.com" value="thecourseforum.com" required>
```

Comment:
Should we do input validation? If you check the `found-a-bug` channel in the Discord you can see where I've made fake bug reports with bogus URLs and emails; not sure how easy it'd be to verify URLs, but emails should be relatively more straightforward

Reply:
Right now the validation only checks if there is any content. For the URLS, I could just set the field so it can't be modified unless you could think of a case when someone might need to change the URL?

Reply:
Honestly, we might be better off making that field optional, or entirely hiding the URL field from the form and just saving it behind the scenes — not sure how many users would either 1) go out of their way to copy a URL just to put in the form or 2) happen to exactly be on the page with the error when they click the bug form.

Another thing to consider is a case where the page crashes when accessing a certain URL, in which case it'd obviously be impossible to access the bug report form from that URL.

Reply:
Maybe create categories like "Crash," "UI/styling error," etc. to help point us in the right direction when looking @ bugs, might be easier than asking them to explicitly give the URL?

Reply:
I just took off the url field but am still sending that info to discord. I think the description part should probably cover all the categories, it might get too busy if we have too many options.

Reply:
yeah, the main idea was just to help us deal w/ reports more efficiently with some categorization, but now that I think about it there probably won't exactly be an overwhelming number of people who go out of their way to do bug reports.

Reply:
I like @alex-shen1's idea of having bug report categories! Users aren't always as descriptive as we'd like them to be.

Having three inputs for "email", "category", and "description" isn't tooo busy (same number as before)

Reply:
Makes sense, I can def add that in. Just thinking that we should limit the number of categories to choose from since if we have one for each feature that might be overwhelming. 

For now will be adding: Crash, UI/Styling, Data Error, Account Error, Other.  @alex-shen1 @j-alicia-long lmk if you have any other ideas/comments for categories.

Reply:
also there's a few options for how to display the categories options
1. multiselect 
<img width="556" alt="multiselect" src="https://user-images.githubusercontent.com/34071587/97316346-04a7a000-1840-11eb-9ef3-8d80aa9ccd43.PNG">
2. select - can only choose 1 option tho
<img width="556" alt="select" src="https://user-images.githubusercontent.com/34071587/97316412-16894300-1840-11eb-8939-70df5f2bdd63.PNG">
3. checkboxes - most similar to write a review mockup, can do horizontal (like below, if everything fits on one line) or vertical
<img width="188" alt="check" src="https://user-images.githubusercontent.com/34071587/97316607-4e908600-1840-11eb-91b6-3aaa6e8b5773.PNG">

Reply:
done :)